editor/src/clj/editor/keymap.clj

```diff
@@ -98,6 +98,7 @@
            ["Meta+'/'" :toggle-comment]
            ["Meta+T" :scene-stop]
            ["Meta+U" :rebundle]
+           ["Meta+J" :close-engine]
```

Comment:
If you have better ideas for a shortcut, you are welcome. 
This shortcut is needed because if we don't add it - it will be annoying to close all the editors...